java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -372,57 +342,6 @@ public T hdel(@NonNull String key, @NonNull String[] fields) {
         return getThis();
     }
 
-    /**
-     * Inserts all the specified values at the tail of the list stored at <code>key</code>.<br>
-     * <code>elements</code> are inserted one after the other to the tail of the list, from the
-     * leftmost element to the rightmost element. If <code>key</code> does not exist, it is created as
-     * an empty list before performing the push operations.
-     *
-     * @see <a href="https://redis.io/commands/rpush/">redis.io</a> for details.
-     * @param key The key of the list.
-     * @param elements The elements to insert at the tail of the list stored at <code>key</code>.
-     * @return Command Response - The length of the list after the push operations.
-     */
-    public T rpush(@NonNull String key, @NonNull String[] elements) {
```

Comment:
Is this just  diff issue?  Or did you move these?

Reply:
Please move new functions to the end to avoid this

Reply:
I am roughly following the order that they had in the Node client and Py. (they differ from each other a bit). This was just a rebase mistake at some point so i just moved it to the correct spot.

Reply:
Since git fails to detect that you added few functions in the middle, you have to reorder them. The actual order does not matter, I added all commends to the end to simplify reviewing.

Reply:
+1